kitsune/wiki/jinja2/wiki/edit_metadata.html

```diff
@@ -0,0 +1,62 @@
+{% extends "wiki/base.html" %}
+{% from "layout/errorlist.html" import errorlist %}
+{% from "wiki/includes/sidebar_modules.html" import document_tools with context %}
+{% from "includes/common_macros.html" import content_editor with context %}
+{% from "wiki/includes/document_macros.html" import edit_messages, submit_revision %}
+{% from "wiki/includes/document_macros.html" import document_lock_warning with context %}
+{% set title = _('Edit Article | {document}')|f(document=document.title) %}
+{# TODO: Change KB url to landing page when we have one #}
+{% set crumbs = [(document.get_absolute_url(), document.title),
+                 (None, _('Edit Article'))] %}
+
+{% set scripts = ('wiki', 'wiki.diff', 'wiki.editor') %}
+{% set classes = 'edit_doc' %}
+
+{% block content %}
+  <div class="user-messages">
+    {{ edit_messages(document, show_revision_warning) }}
+    {{ document_lock_warning() }}
+  </div>
+
+  <div class="sumo-page-section">
+    <article id="edit-document" {% if locked %}class="locked"{% endif %} data-slug="{{ document.slug }}">
+      <h1 class="sumo-page-heading">{{ _('<em>Editing Description For:</em><br>{title}')|fe(title=document.title) }}</h1>
+      {% if document.needs_change %}
+        <p>
+          {% trans comment=document.needs_change_comment %}
+            <label>Needs Change:</label> {{comment}}
+          {% endtrans %}
+        </p>
+      {% endif %}
+      {% if document_form %}
```

Comment:
With your changes in this PR, `document_form` is always defined when using `wiki/edit_metadata.html`, so I think we could get rid of this `{% if document_form %}{% endif %}` wrapper.